data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/DebugUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.scalar.db.dataloader.core.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DebugUtil {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(DebugUtil.class);
```

Comment:
Please use lower camel case for the logger:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(DebugUtil.class);
```

According to the Google Java Style Guide, loggers are not considered constants: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
I have updated the name to `logger`.
Thank you.